js/src/dapps/dappreg/Transactions/transactions.css

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2015-2017 Parity Technologies (UK) Ltd.
+/* This file is part of Parity.
+/*
+/* Parity is free software: you can redistribute it and/or modify
+/* it under the terms of the GNU General Public License as published by
+/* the Free Software Foundation, either version 3 of the License, or
+/* (at your option) any later version.
+/*
+/* Parity is distributed in the hope that it will be useful,
+/* but WITHOUT ANY WARRANTY; without even the implied warranty of
+/* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/* GNU General Public License for more details.
+/*
+/* You should have received a copy of the GNU General Public License
+/* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+.container {
+  bottom: 4rem;
+  position: fixed;
+  right: 0;
+  z-index: 150;
+}
+
+.transaction {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0.5rem;
+  width: 30rem;
+
+  .header {
+    flex: 0 0 auto;
+    height: 1.5rem;
+    position: relative;
+    width: 100%;
+
+    > * {
+      display: inline-block;
+      font-family: 'Roboto Mono', monospace;
+      font-size: 0.8rem;
```

Comment:
Slightly different from the 0.75rem used elsewhere. (Although this is inside a header.)